src/common/Browser.ts

```diff
@@ -27,196 +26,12 @@ import {TaskQueue} from './TaskQueue.js';
 import {TargetManager, TargetManagerEmittedEvents} from './TargetManager.js';
 import {ChromeTargetManager} from './ChromeTargetManager.js';
 import {FirefoxTargetManager} from './FirefoxTargetManager.js';
-
-/**
- * BrowserContext options.
- *
- * @public
- */
-export interface BrowserContextOptions {
-  /**
-   * Proxy server with optional port to use for all requests.
-   * Username and password can be set in `Page.authenticate`.
-   */
-  proxyServer?: string;
-  /**
-   * Bypass the proxy for the given semi-colon-separated list of hosts.
-   */
-  proxyBypassList?: string[];
-}
+import * as BrowserApi from '../api/Browser.js';
```

Comment:
Can be just import what is needed?

Reply:
I don't have a preference but done this mostly to avoid name conflicts and don't assign aliases on case by case basis.